be/src/exec/blocking_join_node.cpp

```diff
@@ -88,6 +88,7 @@ void BlockingJoinNode::build_side_thread(RuntimeState* state, std::promise<Statu
 }
 
 Status BlockingJoinNode::open(RuntimeState* state) {
+    SCOPED_SWITCH_THREAD_LOCAL_MEM_TRACKER(mem_tracker());
```

Comment:
Some `switch` are before `SCOPED_TIMER(_runtime_profile->total_time_counter());`, some are after.
Why are they different?

Reply:
By default `switch` is placed after `SCOPED_TIMER`.
Here, because `switch` needs to be placed before `ExecNode::open`.
Maybe I should also change `SCOPED_TIMER` before `ExecNode::open`.

Reply:
Done,  unite `SCOPED_SWITCH` before `SCOPED_TIMER`.